### PR TITLE
Resources big update

### PR DESCRIPTION
**Pull request by LialinMaxim**
Broken Marshmallow
If you enter valid data, then everything works

### CODE REVIEW

File: src/app/resources.py [hunk 1]
```diff
@@ -9,10 +9,32 @@
...
+user_post = api.model('Create New User', {
+    'email': fields.String,
+    'username': fields.String,
+    'password': fields.String,
+    'first_name': fields.String,
+    'last_name': fields.String,
+    'phone': fields.String,
+})
+
+password_post = api.model('Crate New Password', {
```
Comment (vromanuk) [inline src/app/resources.py]: typo Crate ==> Create

File: src/app/resources.py [hunk 2]
```diff
@@ -77,198 +86,200 @@ def delete(self, *args, **kwargs):
...
 @api.representation('/json')
-class UserListResource(Resource):
+class Home(Resource):
     def get(self):
-        try:
-            users = session.query(User).all()
-        except SQLAlchemyError:
-            return SQLAlchemyError, 500  # Internal Server Error
-        return UserSchema(many=True).dump(users), 200  # OK
+        return 'This is Home Page', 200  # OK
```
Comment (vromanuk) [inline src/app/resources.py]: double quotes or single?

File: src/app/resources.py [hunk 3]
```diff
@@ -77,198 +86,200 @@ def delete(self, *args, **kwargs):
...
-            msg = "User with email: '{0}' is ALREADY EXISTS".format(data['email'])
+            msg = f"User with email: {data['email']} is ALREADY EXISTS"
             return msg, 200  # OK
         else:
-            # TODO optimization USER CLASS
-            # user = User(data)
-            user = User(data['username'], data['email'], data['password'],
-                        data['first_name'], data['last_name'], data['phone'])
-
             # crate a new user
```
Comment (vromanuk) [inline src/app/resources.py]: typo

File: src/app/resources.py [hunk 4]
```diff
@@ -77,198 +86,200 @@ def delete(self, *args, **kwargs):
...
             try:
-                session.add(user)
+                session.add(User(data))
```
Comment (vromanuk) [inline src/app/resources.py]: I think you should leave previsous statement, cause User(data) doesn't seem to be right statement. Or you shold right something like: sesson.add(data), cause I think data is already User model? What do you think?

File: src/app/resources.py [hunk 5]
```diff
@@ -77,198 +86,200 @@ def delete(self, *args, **kwargs):
...
                 session.commit()
-                msg = "New user: '{0}' is SUCCESSFUL ADDED".format(user.username)
-                return msg, 200  # OK
-            except SQLAlchemyError:
-                return SQLAlchemyError, 500  # Internal Server Error
+                return 'NEW USER ADDED', 200  # OK
```
Comment (vromanuk) [inline src/app/resources.py]: In deleted statement we had a username, why did you remove that statement?

File: src/app/resources.py (hunk 4 above)
Comment (vromanuk) [reply src/app/resources.py]: Or tell me what the return value of UserSchema().load?

File: src/app/resources.py [hunk 6]
```diff
@@ -77,198 +86,197 @@ def delete(self, *args, **kwargs):
...
+            return str(err), 500  # Internal Server Error
+
+
+@api.representation('/json')
+class PasswordResource(EntityResource):
+    def get(self, user_id, pass_id):
+        try:
+            if not session.query(User).filter(User.id == user_id).first():
+                return f'User ID {user_id} - Not Found', 404  # Not Found
+            password = session.query(Password) \
```
Comment (vromanuk) [inline src/app/resources.py]: As Vanya said, if you use your code more than 1 time, you have to create a function for that like on lines 253 and 269

Comment (vromanuk) [reply src/app/resources.py]: Add methods into Password model as we have it in User model

File: src/app/resources.py [hunk 7]
```diff
@@ -15,25 +15,25 @@
...
-    'password': fields.String,
-    'first_name': fields.String,
-    'last_name': fields.String,
-    'phone': fields.String,
+    'email': fields.String(example='admin@gmail.com'),
+    'username': fields.String(example='admin'),
+    'password': fields.String(example='admin'),
+    'first_name': fields.String(example='Nicola'),
+    'last_name': fields.String(example='Tesla'),
+    'phone': fields.String(example='068-409-69-36'),
```
Comment (ArturM94) [inline src/app/resources.py]: Do we really need dashes in phone field?
